tests/integration/cli/projects/test/tests/test_fixtures.py

```diff
@@ -12,8 +13,10 @@ def test_chain(chain):
     assert isinstance(chain, ChainManager)
 
 
-def test_contract(Contract):
-    assert isinstance(Contract, ContractInstance)
+def test_contract(accounts):
```

Comment:
I dont see the `Contract` being used?
That is the fixture you are adding, that is what you should be testing.